src/bases/local/laglocal.jl

```diff
@@ -150,7 +150,7 @@ function restrict(f::LagrangeRefSpace{T,1}, dom1, dom2) where T
 
     # for each point of the new domain
     for i in 1:D
-        v = dom2.vertices[i]
+        v = verticeslist(dom2)[i]
```

Comment:
lets rename to vertices. Users might make assumptions on the return data structure if we add list.

Reply:
verticeslist returns a list of vertices, the function vertices in compsciencemshes exists but it does an extra hcat to create a matrix, is this more expensive?